modules/p2/ding_p2_installer/ding_p2_installer.pages_default.inc

```diff
@@ -1,1843 +0,0 @@
-<?php

```

Comment:
I assume that all this is removed because changes have been merged into other page setups.